src/picknik_ur_multi_arm_config/objectives/move_to_pose.xml

```diff
@@ -1,21 +1,18 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <root BTCPP_format="4" main_tree_to_execute="Move to Pose">
   <BehaviorTree ID="Move to Pose" _description="Uses inverse kinematics to move the robot to a set gripper position">
-    <Control ID="Fallback" name="root">
       <Control ID="Sequence">
-        <Action ID="RetrievePoseParameter" timeout_sec="-1" pose="{target_pose}"/>
-        <Action ID="InitializeMTCTask" task_id="move_to_pose" controller_names="/multi_arm_joint_trajectory_controller" task="{move_to_pose_task}"/>
+            <Action ID="InitializeMTCTask" task_id="move_to_pose" controller_names="/multi_arm_joint_trajectory_controller" task="{move_to_pose_task}"/>
         <Action ID="SetupMTCCurrentState" task="{move_to_pose_task}"/>
```

Comment:
```suggestion
        <Action ID="InitializeMTCTask" task_id="move_to_pose" controller_names="/multi_arm_joint_trajectory_controller" task="{move_to_pose_task}"/>
        <Action ID="SetupMTCCurrentState" task="{move_to_pose_task}"/>
```

Reply:
Guess I can't count, thanks for fixing :sweat_smile: